app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -42,7 +42,7 @@
 import com.google.common.collect.Maps;
 
 public class DynamoSurveyDao implements SurveyDao {
```

Comment:
After discussing, I think we are ok with the fact this can cache the same survey 3 times before all paths are cached. In practice 99% of requests will go through one path and the others will not put too much additional load on the system.